src/main/java/net/fabricmc/loader/metadata/ModMetadataV1.java

```diff
@@ -300,6 +300,44 @@ public boolean matches(Version version) {
 							return false;
 						}
 
+						@Override
+						public String getFriendlyVersionString() {
```

Comment:
This implementation could be dragged to an `AbstractModDependency` so we don't have two copies.